jbpm-services/jbpm-services-api/src/build/revapi-config.json

```diff
@@ -42,6 +53,18 @@
         "old": "field org.jbpm.services.api.query.QueryResultMapper<T>.COLUMN_ORGANIZATIONAL_ENTITY",
         "new": "field org.jbpm.services.api.query.QueryResultMapper<T>.COLUMN_ORGANIZATIONAL_ENTITY",
         "justification": "Fix for some of dbs with query aliases"
+      },
+      {
+        "code": "java.field.enumConstantOrderChanged",
+        "old": "field org.jbpm.services.api.query.model.QueryDefinition.Target.CUSTOM",
+        "new": "field org.jbpm.services.api.query.model.QueryDefinition.Target.CUSTOM",
+        "justification": "Included more out of the box targets for query definition that are more specific than custom and thus placed higher in the enum"
+      },
+      {
+        "code": "java.field.serialVersionUIDUnchanged",
+        "old": "field org.jbpm.services.api.query.model.QueryParam.serialVersionUID",
+        "new": "field org.jbpm.services.api.query.model.QueryParam.serialVersionUID",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
```

Comment:
I have done a little bit of a research and this means that the serial version UID stayed unchanged even though the default serial version UID generated by a compiler would be different. I think it might be a good "warning" in the future, but I agree that in this case it probably does not need a change since only a couple of static methods were added. I would agree to add justification like "Not an issue, only static methods were added etc." What do you think?